src/routes/index.js

```diff
@@ -1,18 +1,24 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import cors from 'cors';
+import dotenv from 'dotenv';
 import api from './api/index.route';
 import error from '../middlewares/error.middleware';
 import notfound from '../middlewares/404.middleware';
 
+dotenv.config();
+
 const router = express.Router();
 
 router.use(bodyParser.json());
 router.use(bodyParser.urlencoded({ extended: false }));
 router.use(cors());
+const apiVersion = process.env.API_VERSION;
+
+const baseUrl = `/api/+ ${apiVersion}`;
 
-router.get('/', (req, res) => res.status(200).json({ status: 200, data: 'Welcome to Authors Haven.' }));
-router.use('/api/v1', api);
+router.get('/', (req, res) => res.status(200).json({ status: 200, data: `Welcome to Authors Haven. you are in ${process.env.STATE}` }));
```

Comment:
It would be good to reconcile this route response with what is in the corresponding develop branch

Reply:
> It would be good to reconcile this route response with what is in the corresponding develop branch

hi mike, I was about to test application state where is going to be deployed to different environments it would be easier for us to see when the app is on staging or  production on Heroku

Reply:
Ok, just add the env info as a separate property like {status: ..., data: "welcom...", env: "..."} or something similar so it doesn't fail the test, there's currently only one test and its for this route

Reply:
yeap mike is there in read me as you can see there is STATE tab where you can put development to your local files and it will change vary to different env